### DISCUSSION TITLE
Where are the user abbrevations stored?

### DISCUSSION THREAD

**Discussion by Chekote** [olets/zsh-abbr#119] [Q&A] [answered]
I am trying to erase all of the user abbreviations, but I cannot determine where they are stored. I reviewed the "[Storage and manual editing](https://zsh-abbr.olets.dev/advanced.html#storage-and-manual-editing)" section of the docs, which mentions that a file exists, but not where it is located.

Can someone please help? Thanks 🙇

**Comment by olets** [comment, marked as answer]
The path is stored under `ABBR_USER_ABBREVIATIONS_FILE`. `echo $ABBR_USER_ABBREVIATIONS_FILE` to check yours. Default is documented further up on that page: https://zsh-abbr.olets.dev/advanced.html#configuration-variables.

Would be reasonable to call it out again in the section you linked

**Reply by Chekote** [reply]
Thank you for the prompt answer 🙇

**Reply by olets** [reply]
The docs are updated https://zsh-abbr.olets.dev/advanced.html#storage-and-manual-editing
